src/audio/rtnr/rtnr.c

```diff
@@ -684,8 +684,7 @@ static int rtnr_set_config(struct processing_module *mod, uint32_t param_id,
 #endif
 }
 
-static void rtnr_copy_from_sof_stream(struct audio_stream_rtnr *dst,
-				      struct audio_stream *src)
+void rtnr_copy_from_sof_stream(struct audio_stream_rtnr *dst, struct audio_stream *src)
```

Comment:
these copy can be avoided? if can't, can we use some simple way? like:
memcpy(&dst->size, &src->size, 4*7);

Reply:
ack - we need to keep code as-is and only touch the parts that interface with the module API to minimize any impact to the validated code.